ch2driver/pytpcc/drivers/abstractdriver.py

```diff
@@ -66,6 +66,8 @@ def formatConfig(self, config):
     def getOneDoc(self, tableName, fieldValues, generateKey=False):
         if self.schema == constants.CH2_DRIVER_SCHEMA["CH2"]:
             return self.getOneCH2Doc(tableName, fieldValues, generateKey)
+        elif self.schema == constants.CH2_DRIVER_SCHEMA["CH2PPF"]:
+            return self.getOneCH2PPFlatDoc(tableName, fieldValues, generateKey)
```

Comment:
How are we handling CH2PF? Should that not also use `getOneCH2PPFlatDoc`?

Reply:
We will not have a CH2PF variant.